src/Not/Not.Blazor/Components/Mud/NSimpleTable.razor.cs

```diff
@@ -11,6 +15,20 @@ public partial class NSimpleTable<T>
     [Parameter]
     public string EmptyMessage { get; set; } = "This table is still empty";
 
+    [Parameter]
+    public string Height { get; set; } = default!;
+
+    [Parameter]
+    public bool Mobile { get; set; } = false;
+
     [Parameter]
     public RenderFragment<T> CustomCell { get; set; } = default!;
+
+    protected override void OnParametersSet()
+    {
+        if (Mobile)
+        {
+            _typography = Typo.caption;
+        }
```

Comment:
This should be done on theme level, not on component level. Params like `Mobile` or `IsMobile` should be avoided in application code.